packages/sdk/src/MidasSdk/index.ts

```diff
@@ -98,11 +100,45 @@ export class MidasBase {
     return this._signer;
   }
 
+  public get contracts(): {
```

Comment:
This will likely require changing all the bots code, AFAICT

Reply:
What's the reasoning here btw?

Reply:
probably not, it's a getter function. `sdk.contract.FusePoolLens` works as before, no worries.

I had to do it as somehow the instance created in the constructor gets out of sync and is not updated. I don't like this implementation as well, but it solves this issue. Wasn't able to find a better solution in a reasonable amount of time.